chainer/functions/array/pad.py

```diff
@@ -0,0 +1,65 @@
+import numpy
+
+from chainer import cuda
+from chainer import function
+from chainer.utils import type_check
+
+
+class Pad(function.Function):
+
+    """Padding of an array"""
+
+    def __init__(self, pad_width, mode, **keywords):
+        self.mode = mode
+        self.keywords = keywords
+        self.pad_width = pad_width
+        self.pad_bw = numpy.asarray(pad_width)
+        if self.pad_bw.size == 1:
+            self.pad_bw = numpy.repeat(self.pad_bw, 2)
+
+    def check_type_forward(self, in_types):
+        type_check.expect(in_types.size() == 1)
+
+    def forward(self, inputs):
+        xp = cuda.get_array_module(*inputs)
+        if len(self.keywords) == 0:
+            return xp.pad(inputs[0], self.pad_width, mode=self.mode),
+        else:
+            return xp.pad(inputs[0], self.pad_width, mode=self.mode,
+                          **self.keywords),
```

Comment:
I thought it works even if `self.keywords` is an empty dictionary and we do not need the branching. Doesn't it work?